client/packages/system/src/Item/api/hooks/useDefaultServiceItem/useDefaultServiceItem.tsx

```diff
@@ -3,9 +3,10 @@ import { useServiceItems } from '../useServiceItems';
 export const useDefaultServiceItem = () => {
   const { data: items, isLoading, error } = useServiceItems();
 
-  const defaultServiceItem = items?.nodes.find(
-    ({ code }) => code === 'service'
-  );
+  // find the first service item with the code 'service'
+  // or if that fails, simply the first service item
+  const defaultServiceItem =
+    items?.nodes.find(({ code }) => code === 'service') ?? items?.nodes?.[0];
```

Comment:
yes it can. this is just the 'default' item - when you click add, a new row is added to the table and editing enabled, at which point you can select a different service item. however, in order to add a valid row, you need to have a service item, so we want to have one to pick by default. This code is just to pick out the default - and mSupply doesn't have the concept of a 'default' service item, so we were just taking the one with the code 'service' which was fairly standard at the time.

Was brought in with [this PR](https://github.com/msupply-foundation/openmsupply-client/pull/1021/files#diff-0dba222948e19497031a21db0e6a3f56823a3c5b254e07c7469cc7d5fde70713)

Reply:
Ah right, sorry, misread something